main.py

```diff
@@ -2,6 +2,42 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    # CREATE
```

Comment:
this comment doesn't provide any additional info, so it is redundant. If you wanna separate blocks of code somehow - use blank lines.
Fix everywhere